python/cac_tripplanner/destinations/models.py

```diff
@@ -15,7 +15,7 @@ class DestinationManager(models.GeoManager):
     """Custom manager for Destinations allows filtering on published"""
 
     def published(self):
-        return self.get_queryset().filter(published=True)
+        return self.get_queryset().filter(published=True).order_by('priority')
```

Comment:
It may be a distinction without a difference, but I'd be inclined to set this as the default ordering for the model, i.e.
```
class Meta:
    ordering = ['priority']
```
If we want to keep randomizing destinations with tied priorities, it could be `ordering = ['priority', '?']`

Also, not directly related but since we're right here, you could delete `get_queryset` since it's just calling super with no modification.